examples/pingpong/pong.js

```diff
@@ -1,12 +1,16 @@
-var Queue = require('../../');
+/* eslint-disable no-console */
+'use strict';
 
-var pingQueue = Queue('ping');
-var pongQueue = Queue('pong');
+const Queue = require('../../');
 
-pingQueue.process(function (job, done) {
-  console.log('Pong received ping');
-  pongQueue.createJob().save(function () {
-    console.log('Pong sent back pong');
-    done();
-  });
+const pingQueue = new Queue('ping');
+const pongQueue = new Queue('pong');
+
+pingQueue.process(async (job) => {
+  console.log(`Ping job received: (ID: ${job.id})`);
+
+  await pongQueue.createJob().save();
```

Comment:
I think it'd be neat to include the received job ID in the data sent over the pong queue to add more understanding-hooks.